leakcanary-android-core/build.gradle

```diff
@@ -1,5 +1,6 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
+apply plugin: 'com.vanniktech.maven.publish'
```

Comment:
these changes create conflicts because we've changed the setup in all build scripts.